.github/CONTRIBUTING.md

```diff
@@ -38,13 +44,21 @@ We work hard to process bugs that are reported, to assist with this please ensur
 
 If you find out your bug is actually a duplicate of another bug and only notice that after you created it, please also close your bug with a short reference to the other issue that was there before.
 
-#### Reporting security issues
+#### Reporting an issue with an October plugin
```

Comment:
Should this be linked in the ToC?

Reply:
@LukeTowers It's more an informational section, not really a "contribution" action that people can take, so that's why I kept it out of the ToC.

Reply:
I know, but it's something that people might be looking for so we might as well include it in the ToC. Thoughts?

Reply:
I'd be happy to make it a "sub-link" underneath "Reporting an issue with OctoberCMS". Also, it is currently the first **Note** mentioned underneath that section too:
https://github.com/octobercms/october/blob/9ee6bb69b42fab449db33ce3cb9b41c30f9980c8/.github/CONTRIBUTING.md#reporting-an-issue-with-octobercms

Reply:
Works for me @bennothommo

Reply:
Done :)